### PR TITLE
52 load data without positioner

### LINKED ISSUE #52: Load data without a positioner
```bash
Traceback (most recent call last):
  File "/home/beams22/29IDUSER/bin/mdaviz/mdaviz/mda_folder_table_model.py", line 60, in data
    return action(file)
           ^^^^^^^^^^^^
  File "/home/beams22/29IDUSER/bin/mdaviz/mdaviz/mda_folder_table_model.py", line 24, in <lambda>
    "Positioner": lambda file: self.get_file_pos(file),
                               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/beams22/29IDUSER/bin/mdaviz/mdaviz/mda_folder_table_model.py", line 99, in get_file_pos
    pv = utils.byte2str(readMDA(str(filepath))[1].p[0].name)
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range
```

### LINKED ISSUE #54: Limit the Number of Maximum Subfolders Processed in get_all_subfolders Function
The `get_all_subfolders` function currently processes all subfolders in a selected directory without restriction. This can lead to performance issues when the function encounters directories with a large number of subfolders or a deep directory structure. Should limit both the number of subfolders and the directory depth, ideally configurable via user settings with sensible default settings. 

- **Depth Limit**: Users can choose how deep the program goes into folder layers.
- **Subfolder Count Limit**: Users can also decide on the maximum number of subfolders the program should check at each level. 
- **User Feedback**: Inform the user when the depth or subfolder count limit is reached, indicating that some subfolders might not have been processed.

### PR DESCRIPTION
**Pull request by rodolakis**
- close #52 
- close #54

### CODE REVIEW

File: env.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ dependencies:
 
     # additional dependencies of the file directory support additions
     # install from GitHub repository
-    - "https://raw.githubusercontent.com/EPICS-synApps/support/master/utils/mdaPythonUtils/mdalib-2022.post9-py2.py3-none-any.whl"
\ No newline at end of file
+    - https://github.com/EPICS-synApps/support/blob/master/utils/mdaPythonUtils/mda.py
```
Comment (prjemian) [inline env.yml:29]:
```suggestion
    - https://raw.githubusercontent.com/EPICS-synApps/support/master/utils/mdaPythonUtils/mdalib-2022.post9-py2.py3-none-any.whl
```

Comment (prjemian) [reply env.yml:29]: Need the raw content, otherwise it will download the HTML-formatted page.

Comment (rodolakis) [reply env.yml:29]: Isn't it what I had initially. I misunderstood your comment [here](https://github.com/BCDA-APS/mdaviz/issues/52#issuecomment-1922273161)

Comment (prjemian) [reply env.yml:29]: Yes, it is.

**Review by prjemian — APPROVED** [review]
LGTM

**Pull request merged**